File: duplicate-assert/python-unittest/XpathUtilTest_testGetValue.py
import unittest

class XpathUtilTest(unittest.TestCase):

    def test_get_value(self):
        self.assertEqual("HELLO WORLD", get_text_attribute_value(
            create_detail_ast(TokenTypes.STRING_LITERAL, "\"HELLO WORLD\"")),
            "Returned value differs from expected")
        self.assertEqual("123", get_text_attribute_value(
            create_detail_ast(TokenTypes.NUM_INT, "123")),
            "Returned value differs from expected")
        self.assertEqual("HELLO WORLD", get_text_attribute_value(
            create_detail_ast(TokenTypes.IDENT, "HELLO WORLD")),
            "Returned value differs from expected")
        self.assertNotEqual("HELLO WORLD", get_text_attribute_value(
            create_detail_ast(TokenTypes.STRING_LITERAL, "HELLO WORLD")),
            "Returned value differs from expected")

File: duplicate-assert/java-junit/XpathUtilTest_testGetValue.java
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotEquals;

import org.junit.jupiter.api.Test;

public class XpathUtilTest {

    @Test
    public void testGetValue() {
        assertEquals("HELLO WORLD", getTextAttributeValue(
                createDetailAST(TokenTypes.STRING_LITERAL, "\"HELLO WORLD\"")),
                "Returned value differs from expected");
        assertEquals("123", getTextAttributeValue(createDetailAST(TokenTypes.NUM_INT, "123")),
                "Returned value differs from expected");
        assertEquals("HELLO WORLD",
                getTextAttributeValue(createDetailAST(TokenTypes.IDENT, "HELLO WORLD")),
                "Returned value differs from expected");
        assertNotEquals("HELLO WORLD",
                getTextAttributeValue(createDetailAST(TokenTypes.STRING_LITERAL, "HELLO WORLD")),
                "Returned value differs from expected");
    }
}
